upgrading/version_specific_migration/v60.md

```diff
@@ -0,0 +1,190 @@
+# Upgrade to Plone 6.0
+
+Plone 6.0 has seen the following major changes.
+Some may require changes in your setup.
+
+## Removed portal_quickinstaller
+Plone no longer ships with the `portal_quickinstaller` tool (`CMFQuickInstallerTool`).
+In existing sites, the standard upgrade will remove the tool.
+This is the final step in a deprecation path that started in Plone 5.1.
+Documentation on how to switch to GenericSetup and the new installer, can be seen in the old [Plone 5.1 upgrade guide](
+https://docs.plone.org/develop/addons/upgrade_to_51.html#installation-code).
+
+See [PLIP 1775](https://github.com/plone/Products.CMFPlone/issues/1775).
+
+
+## Dexterity site root
+For contenttypes, in Plone 4 and 5 you had the option to use the old Archetypes or the new Dexterity framework.
```

Comment:
I think it should be `content types` see: https://fsec404.github.io/blog/HTTP-parameter-pollution/#parameter-pollution-in-social-share-buttons
```suggestion
For content types, in Plone 4 and 5 you had the option to use the old Archetypes or the new Dexterity framework.
```